util/dvsim.py

```diff
@@ -352,7 +352,7 @@ def main():
         "-mp",
         "--max-parallel",
         type=int,
-        default=32,
+        default=16,
```

Comment:
What's this change for? Is it related to usbdev?

Reply:
This is the DV regression tool that spawns simulations in parallel. 32 forked process was causing too much strain on the resources (losing the ability to move even the mouse), so I reduced it to 16. This tool is fairly new in the process of wider team deployment. It does not affect a lot of people yet.

Reply:
I removed this change - will make it a part of a separate PR.

Reply:
Thanks for clarifying and separate out this change.